### DISCUSSION TITLE
Read .prf-file in MIKE IO 1D v0.3

### DISCUSSION THREAD

**Discussion by didnil** [DHI/mikeio1d#49] [Q&A] [answered]
Hi!

In mikeio1d 0.1 it was possible to read .prf-files, which I haven't been able to do in 0.3. Is there another way for doing it, or will it be possible in a future release? It is very helpful when comparing a converted Mike+ model with the original Mike urban model.

**Comment by gedaskir** [comment, marked as answer]
Hello, it is a bug that was introduced in v0.3. I have made a potential fix in a bug fix branch. Would it be possible for you to try it out?

To install the bug fix branch do the following:

1. Uninstall MIKE IO 1D:
```cmd
pip uninstall mikeio1d
```

2. Install MIKE IO 1D from bug fix branch:
```cmd
pip install https://github.com/DHI/mikeio1d/archive/bug/read_mouse_results.zip
```

**Reply by didnil** [reply]
The solution works, thank you! And thanks for maintaining this very helpfull package.
